src/test/models/Comment.test.js

```diff
@@ -14,6 +15,18 @@ describe('Test for Comment Model', () => {
   checkModelName(Comment)('Comments');
 
   context('properties', () => {
-    ['comment'].forEach(checkPropertyExists(comment));
+    ['comment', 'userId', 'requestId'].forEach(checkPropertyExists(comment));
```

Comment:
nice @torsami77 , i think you will have to just use `comment` here for the test to finally pass

Reply:
Thanks bro, have made the changes